[CMSP-954] New PHP Defaults (#9111)
pantheon-systems/documentation: source/releasenotes/2024-07-17-php-defaults.md

---
title: New PHP Defaults
published_date: "2024-07-17"
categories: [infrastructure, wordpress, drupal, action-required]
---

> Pantheon has pushed an update to the WordPress, Drupal 7, and Drupal Composer Managed upstreams which sets PHP 8.2 as the default PHP version, rather than 8.1.

Reviewer: ```suggestion
Pantheon has pushed an update to the WordPress, Drupal 7, and Drupal Composer Managed upstreams which sets the default PHP version to 8.2 (previously PHP 8.1).
```
